README.md

```diff
@@ -1,55 +1,22 @@
 # schwifty-migration
```

Comment:
Controversial question :P do we still think this is a good package name?

Reply:
Package name is definitely up for review -- any thoughts?

Reply:
Maybe we should decide based upon the name of the command?